terraform/eks/main.tf

```diff
@@ -108,6 +111,18 @@ resource "kubernetes_service_account" "aoc-role" {
   automount_service_account_token = true
 }
 
+resource "kubernetes_service_account" "aoc-fargate-role" {
+  metadata {
+    name      = "aoc-fargate-role-${module.common.testing_id}"
+    namespace = "default"
+    annotations = {
+      "eks.amazonaws.com/role-arn" : "arn:aws:iam::611364707713:role/ServiceAccount-eks-test-aoc-role"
```

Comment:
Can we set the account Id to a dynamic one instead of default one?

Reply:
This I was going to change in the next phase.

Reply:
Got cha. Should  catch up with the plan for the current phase.